### PR TITLE
Qc check rnaseqc

### PR DESCRIPTION
**Pull request by rube14**
All changes are about RNA-SeQC template in SeQC. Requires rna_seqc table in npg_qc db (npgqctest_rb11 schema in npgtest-db can be used).

### CODE REVIEW

File: lib/npg_qc/autoqc/results/rna_seqc.pm [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+                                      end_2_pct_sense
+                                      mean_per_base_cov
+                                      mean_cv
+                                      end_5_norm
+                                      end_3_norm
+                                    /;
+
+has [ @ATTRIBUTES ] => (
+    is         => 'rw',
+    isa        => 'Maybe[Num]',
```
Comment (mgcam) [inline lib/npg_qc/autoqc/results/rna_seqc.pm]: Please see https://github.com/wtsi-npg/npg_irods/issues/89

File: data/schema.txt [hunk 2]
```diff
@@ -1131,6 +1131,40 @@ CREATE TABLE `ref_snp_info` (
...
+-- Table structure for table `rna_seqc`
+--
+
+DROP TABLE IF EXISTS `rna_seqc`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `rna_seqc` (
+  `id_rna_seqc` bigint(20) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Auto-generated primary key',
+  `id_seq_composition` bigint(20) unsigned NOT NULL COMMENT 'A foreign key referencing the id_seq_composition column of the seq_composition table',
+  `path` varchar(256) DEFAULT NULL,
```
Comment (mgcam) [inline data/schema.txt]: I am not sure that this field is going to be filled by this check. This is a historic feature, which we probably can discontinue.

**Comment by mgcam** [comment]
A file with an SQL statement that will update a database is needed in https://github.com/wtsi-npg/npg_qc/tree/devel/scripts/upgrade_schema

File: lib/npg_qc/autoqc/results/rna_seqc.pm (hunk 1 above)
Comment (dkj) [inline lib/npg_qc/autoqc/results/rna_seqc.pm]: Would like to chat this particular case over. I think important here are
- reflecting what data we are capturing and 
- consistency serializing to the DB.

File: scripts/upgrade_schema/upgrade_schema-60.x [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  `end_2_antisense` float unsigned DEFAULT NULL,
+  `end_1_pct_sense` float unsigned DEFAULT NULL,
+  `end_2_pct_sense` float unsigned DEFAULT NULL,
+  `mean_per_base_cov` float unsigned DEFAULT NULL,
+  `mean_cv` float unsigned DEFAULT NULL,
+  `end_5_norm` float unsigned DEFAULT NULL,
+  `end_3_norm` float unsigned DEFAULT NULL,
+  `other_metrics` text,
+  PRIMARY KEY (`id_rna_seqc`),
+  KEY `rna_seqc_compos` (`id_seq_composition`),
```
Comment (ces) [inline scripts/upgrade_schema/upgrade_schema-60.x]: needs updating now - change to;
UNIQUE KEY `rna_seqc_id_compos_unq` (`id_seq_composition`),

**Pull request merged**